spec/controllers/users_controller_spec.rb

```diff
@@ -203,6 +203,13 @@
         expect(Workers::Mail::ConfirmEmail).to receive(:perform_async).with(@user.id).once
         put(:update, :id => @user.id, :user => { :email => "my@newemail.com"})
       end
+
+      it "saves unconfirmed_email when podmail is disabled" do
+        AppConfig.mail.enable = false
+        put(:update, :id => @user.id, :user => { :email => "my@newemail.com"})
```

Comment:
Want me to fix the rest of the tests in that file? There's a number of them using the older syntax (I had just copied one of the existing tests and modified)